packages/utils/src/spam-filter/spam-filter.spec.ts

```diff
@@ -5,16 +5,21 @@ describe('Spam filter', () => {
     expect(spamFilter('This token name is OK')).not.toEqual(spamReplacement);
   });
   it('should detect spam urls in strings and replace content', () => {
-    expect(spamFilter('www.fake')).toEqual(spamReplacement);
-    expect(spamFilter('https://www.fake.com')).toEqual(spamReplacement);
+    expect(spamFilter('www.fake')).toEqual('www.fake');
```

Comment:
This, and all the other tests changed below, should be removed because they no longer reflect the functionality described on L7